### PR TITLE
Changed .global_driveignore location to OS config dir

### LINKED ISSUE #1: Move .global_driveignore to OS' config folder
Currently the `.global_driveignore` is stored next to the downloaded source files. I would like it to be moved to `$config/driveignore/.global_driveignore` where `$config` is the OS' config folder. Determine it with `os.UserConfigDir()`.

### PR DESCRIPTION
**Pull request by jketterer**
Fixes #1 and adds some error catching

### CODE REVIEW

File: cmd/global.go [hunk 1]
```diff
@@ -35,18 +34,26 @@ Once you set your global .driveignore you can delete the file you pointed to.`,
...
 		content, _ := ioutil.ReadFile(absPath)
 		vPrint("loaded file contents")
 		if shouldAppend {
-			currContent, _ := ioutil.ReadFile(filepath.Join(currFile, "../../.global_driveignore"))
-			ioutil.WriteFile(filepath.Join(currFile, "../../.global_driveignore"), []byte(string(content)+"\n"+string(currContent)), os.ModePerm)
-			vPrint("appended the contents to the .global_driveignore")
+			currContent, _ := ioutil.ReadFile(filepath.Join(configDir, ".global_driveignore"))
+			err := ioutil.WriteFile(filepath.Join(configDir, ".global_driveignore"), []byte(string(content)+"\n"+string(currContent)), os.ModePerm)
+			if err != nil {
+				vPrint(err)
```
Comment (shilangyu) [inline cmd/global.go]: vPrint is for 'verbose print' therefore the error will never be printed if verbose isnt on. Instead change the function `Run` to `RunE` and return the error.

File: cmd/global.go [hunk 2]
```diff
@@ -35,18 +34,26 @@ Once you set your global .driveignore you can delete the file you pointed to.`,
...
+			} else {
+				vPrint("appended the contents to the .global_driveignore")
+			}
 		} else {
-			ioutil.WriteFile(filepath.Join(currFile, "../../.global_driveignore"), content, os.ModePerm)
-			vPrint("saved the contents to the .global_driveignore")
+			err := ioutil.WriteFile(filepath.Join(configDir, ".global_driveignore"), content, os.ModePerm)
+			if err != nil {
+				vPrint(err)
```
Comment (shilangyu) [inline cmd/global.go]: same here

**Review by shilangyu — CHANGES_REQUESTED** [review]
After this small change ill accept the PR. Thanks for contributing!

**Comment by shilangyu** [comment]
@jketterer do you plan to make these changes?

**Comment by jketterer** [comment]
@shilangyu The latest commit should fix the problem you're referring to, let me know if I missed something though.

**Review by shilangyu — APPROVED** [review]

**Comment by shilangyu** [comment]
Thanks for contributing!

**Pull request merged**